mne/viz/raw.py

```diff
@@ -785,6 +795,20 @@ def _plot_raw_traces(params, color, bad_color, event_lines=None,
                 this_decim = 1
             this_t = params['times'][::this_decim] + params['first_time']
 
+            # clip to range (if relevant)
+            if params['clipping'] == 'clamp':
+                np.clip(this_data, -1, 1, out=this_data)
+            elif params['clipping'] is not None:
+                l, w = this_t[0], this_t[-1] - this_t[0]
+                ylim = params['ax'].get_ylim()
+                b = offset - params['clipping']  # max(, ylim[0])
+                h = 2 * params['clipping']  # min(, ylim[1] - b)
+                assert ylim[1] <= ylim[0]  # inverted
```

Comment:
Why is this inverted..?

Reply:
It's just internally how we plot these, I don't remember the reasons...

Reply:
Hahaha ok fair enough 😅